src/pretalx/orga/views/mails.py

```diff
@@ -69,7 +82,13 @@ def dispatch(self, request, *args, **kwargs):
         return redirect(self.request.event.orga_urls.outbox)
 
 
-class OutboxPurge(View):
+class OutboxPurge(PermissionRequired, View):
+    permission_required = 'orga.view_mails'
```

Comment:
There is `orga.purge_mails`, which seems more fitting here?

Reply:
Yup.